packages/codegen-ui/lib/__tests__/generate-form-definition/generate-form-definition.test.ts

```diff
@@ -398,6 +413,7 @@ it('should add read-only fields if it has overrides', () => {
     name: {
       componentType: 'TextField',
       dataType: 'String',
+      isArray: false,
```

Comment:
I think you have to let the metadata get its `isArray` field from the form def element now, or it won't get picked up during codegen. The change is in `form-component-metadata.ts`

Reply:
discussed offline. change was not needed based on previous implementation, but a new version was pushed which no longer relies on referencing dataSchema.